app/controllers/quotes_controller.rb

```diff
@@ -35,7 +35,7 @@ def create
   end
 
   def index
-    @quotes = Quote.all
+    @quotes = Quote.order("created_at DESC")
```

Comment:
on new versions of rails I think `.order(created_at: :desc)` is preferred, or something like that